### PR TITLE
add purge call

### LINKED ISSUE #22: PURGE
@mshmsh5000 says we should probably add a `PURGE` to the service for updates and deletes.

Currently the tables reflect what we want them to, but fastly caches stuff, so if you delete a redirect it will still work.

https://docs.fastly.com/api/purge

### PR DESCRIPTION
**Pull request by katiecrane**
Fixes #22 
- add calls to purge fastly cache for individual urls for whatever service nightwing is working with (which will eventually be phoenix prod, currently thor)
- the domain that the fastly service works on now must be specified in the `.env` because:
  - the only way to get the domain via the fastly api is to get the active version of the service and then use that to get the domain. There is no call to just get the active version of the service, it would have to be a call to get ALL versions and then parse through to see which one is tagged as active. I suppose that is possible but since the service key has to be specified in the `.env` anyway, seems to make sense to just toss the domain in there as well.
  - we need the domain in the first place in order to purge an individual url. the other purge options are to purge everything (which is overkill plus they suggest doing that only once a week) or to use surrogate keys (which doesn't seem like the right approach for this since we really do want to purge individual urls)
- I tested this with microwookie.com and the updates are instant! It's pretty cool!

cc: @mshmsh5000

### CODE REVIEW

File: app/Providers/FastlyProvider.php [hunk 1]
```diff
@@ -96,13 +97,12 @@ public function boot()
              Log::info(sprintf('Response: call 2: %s',
                 $response));
 
-            // Step three: Purge this service so the deleted redirect stops working immediately
+            // Step three: Purge the url of the deleted redirect so the deleted redirect stops working immediately
```
Comment (mshmsh5000) [inline app/Providers/FastlyProvider.php:99]: Seems as though you could DRY this and the other block into a single "purge" method.

Comment (katiecrane) [reply app/Providers/FastlyProvider.php:99]: oh yeah, that's a good idea! thanks!

File: .env.example [hunk 2]
```diff
@@ -22,10 +22,12 @@ MAIL_USERNAME=null
...
 NORTHSTAR_API_KEY=abc4324
 
 FASTLY_KEY=abc123
 FASTLY_TABLE_REDIRECTS=abc123
 FASTLY_TABLE_REDIRECT_TYPES=abc123
-FASTLY_SERVICE_KEY=abc123
\ No newline at end of file
+FASTLY_SERVICE_KEY=abc123
+
+SERVICE_DOMAIN=example.com
```
Comment (angaither) [inline .env.example]: this is still fastly, right? might wanna just name it that, or add a comment for clarity

**Comment by angaither** [comment]
one note, otherwise 🚢

File: .env.example (hunk 2 above)
Comment (katiecrane) [reply .env.example]: @angaither yeah it's the domain that the fastly service works on (so eventually dosomething.org). Is `FASTLY_SERVICE_DOMAIN` clearer?

**Comment by mshmsh5000** [comment]
Hey this is nice! 🐑 🐑 🐑 (After the tests are happy)

**Comment by katiecrane** [comment]
according to @blisteringherb it's expected that wercker fails because it's not actually deploying from there!

**Pull request merged**